dart/lib/src/sentry_options.dart

```diff
@@ -184,6 +184,43 @@ class SentryOptions {
   /// sent. Events are picked randomly. Default is null (disabled)
   double? sampleRate;
 
+  /// The ignoreErrors tells the SDK which errors should be not sent to the sentry server.
+  /// If an null or an empty list is used, the SDK will send all transactions.
+  Iterable<String>? ignoreErrors;
```

Comment:
Let's use `List`, I don't think it really matters much but we can keep it consistent with the rest of the code